src/Components/Order/OrderInfo.tsx

```diff
@@ -0,0 +1,208 @@
+import styled from '@emotion/styled';
+import { useOrderForm } from '@/hooks/useOrderForm';
+import { useNavigate } from 'react-router-dom';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  padding: 20px 0;
+  border-radius: 8px;
+`;
+
+const Section = styled.div`
+  padding: 20px 16px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  border-radius: 8px;
+`;
+
+const Label = styled.label`
+  font-size: 14px;
+  font-weight: 600;
+`;
+
+const InputBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+`;
+
+const StyledInput = styled.input<{ error?: boolean }>`
+  width: 100%;
+  height: 44px;
+  padding: 0 12px;
+  font-size: 14px;
+  border: 1px solid ${({ error, theme }) => (error ? 'red' : theme.color.semantic.border.default)};
+  border-radius: 8px;
+  outline: none;
+
+  &::placeholder {
+    color: ${({ theme }) => theme.color.semantic.text.placeholder};
+  }
+`;
+
+const ErrorMsg = styled.p`
+  color: ${({ theme }) => theme.color.semantic.status.critical};
+  font-size: 12px;
+  margin: 0;
+`;
+
+const HelperText = styled.span`
+  font-size: 12px;
+  color: ${({ theme }) => theme.color.semantic.text.placeholder};
+`;
+
+const ProductInfo = styled.div`
+  padding: 20px 16px;
+  border-radius: 8px;
+  display: flex;
+  gap: 12px;
+  align-items: center;
+`;
+
+const ProductImage = styled.img`
+  width: 64px;
+  height: 64px;
+  object-fit: cover;
+  border-radius: 4px;
+`;
+
+const ProductDetails = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+
+  & > strong {
+    font-size: 14px;
+  }
+
+  & > span {
+    font-size: 12px;
+    color: ${({ theme }) => theme.color.semantic.text.placeholder};
+  }
+
+  & > b {
+    font-weight: bold;
+  }
+`;
+
+const OrderButton = styled.button`
+  width: 100%;
+  height: 56px;
+  background-color: ${({ theme }) => theme.color.semantic.brand.kakaoYellow};
+  border: none;
+  font-size: 16px;
+  font-weight: bold;
+  border-radius: 4px;
+  cursor: pointer;
+`;
+
+const GiftForm = () => {
+  const { values, errors, handleChange, validate } = useOrderForm({
+    sender: '',
+    receiver: '',
+    phone: '',
+    quantity: 1,
+    message: '',
+  });
+
+  const navigate = useNavigate();
+  const handleSubmit = () => {
+    if (validate()) {
+      alert(
+        `주문이 완료되었습니다.
+        상품명: BBQ 양념치킨+크림치즈볼+콜라1.25L
+        구매 수량: ${values.quantity}
+        발신자 이름: ${values.sender}
+        메시지: ${values.message}`
+      );
+      navigate('/');
+    }
+  };
```

Comment:
이럴 때는 아래처럼 early return을 활용하시면 코드가 더 간결해지고,
depth가 줄어들어 가독성이 좋아집니다~

```suggestion
  const handleSubmit = () => {
    if (!validate()) return;
    
    alert(
      `주문이 완료되었습니다.
      상품명: BBQ 양념치킨+크림치즈볼+콜라1.25L
      구매 수량: ${values.quantity}
      발신자 이름: ${values.sender}
      메시지: ${values.message}`
    );
    navigate('/');
  };
```